msi/GitUI_en-us.wxl

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
+<WixLocalization Culture="en-us" xmlns="http://wixtoolset.org/schemas/v4/wxl">
+  <!-- TODO: Change the ID's to be prefixed with 'Git' instead of 'GfwUI_' -->
+  <!-- Bitmap and Icon Binary Table Reference -->
+  <String Id="GfwUI_CompleteSetup_Icon" Overridable="yes">completi</String>
```

Comment:
I ought to clean this up before we merge. I'll take care of this tomorrow. These either aren't needed anymore _or_ I should rename them and get rid of the `<!-- TODO ... -->` comment.